source/tutorials/nix-language.md

```diff
@@ -109,14 +109,13 @@ The following example is a Nix expression adding two numbers:
 
 #### Interactive evaluation
 
-Use [`nix repl`] to evaluate Nix expressions interactively (by typing them on the command line):
-
-```shell-session
-$ nix repl
-Welcome to Nix 2.13.3. Type :? for help.
+Use the Nix REPL to evaluate Nix expressions interactively (by typing them on the command line).
+Execute it as `$ nix repl`, and you have a repl:
```

Comment:
That is not an improvement because now there is no link to reference docs. It was good as it was.